recipes/txl-local-terminals/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+noarch_platform:
```

Comment:
Please see https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies for the correct filename and contents.